src/main/java/site/coach_coach/coach_coach_server/category/service/CategoryService.java

```diff
@@ -44,10 +46,18 @@ public void deleteCategory(Long routineId, Long categoryId, Long userIdByJwt) {
 	}
 
 	@Transactional
-	public Category changeIsCompleted(Long categoryId, Long routineId) {
+	public Category changeIsCompleted(Long categoryId, Long routineId, Boolean inputIsCompleted) {
 		Category category = categoryRepository.findByCategoryIdAndRoutine_RoutineId(categoryId, routineId)
 			.orElseThrow(() -> new NotFoundCategoryException(ErrorMessage.NOT_FOUND_CATEGORY));
 
+		if (category.getIsCompleted().equals(inputIsCompleted)) {
+			if (inputIsCompleted) {
+				throw new DuplicateCompletedCategoryException(ErrorMessage.DUPLICATE_COMPLETED_CATEGORY);
+			} else {
+				throw new NotFoundCompletedCategoryException(ErrorMessage.NOT_FOUND_COMPLETED_CATEGORY);
+			}
+		}
+
 		// Dirty Checking
 		category.setIsCompleted(!category.getIsCompleted());
```

Comment:
```suggestion
		category.setIsCompleted(inputIsCompleted)
```
이렇게 할 수도 있지 않을까용? 데이터를 꺼내오는 일이 적어서 더 좋을 것 같아요!

Reply:
요거는 inputIsCompleted가 사라지면서 기존 코드를 유지하도록 하겠습니다!

Reply:
그렇군요,, 사실 저게 왜있나 싶긴했습니다..하하!